server/content-security-policy.js

```diff
@@ -56,6 +56,7 @@ const COMMON_DIRECTIVES = {
     '*.paypal.com',
     '*.paypalobjects.com',
     'https://hcaptcha.com',
+    'https://js.hcaptcha.com',
```

Comment:
@kewitz Why was this not catch by the catchall below?

Reply:
It was actually working locally, I thought it was something related to E2E test environment.
I just double-checked the test and it seems it was just a flaky E2E test, it was safe to merge since the beginning. 😒